From jeffm@suse.com  Mon Aug 30 16:56:28 2010
Message-ID: <4C73EF6A.5050603@suse.com>
Date: Tue, 24 Aug 2010 12:12:26 -0400
From: Jeff Mahoney <jeffm@suse.com>
To: Greg Kroah-Hartman <GregKH@suse.de>
Subject: Staging: spectra: depend on X86_MRST

lld_nand fails to build on arches without virt_to_bus. Since this driver
is specifically for hardware enablment on Moorestown, this patch adds
Moorestown MID support as a dependency.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Cc: David Woodhouse <dwmw2@infradead.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/spectra/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/spectra/Kconfig
+++ b/drivers/staging/spectra/Kconfig
@@ -2,6 +2,7 @@
 menuconfig SPECTRA
 	tristate "Denali Spectra Flash Translation Layer"
 	depends on BLOCK
+	depends on X86_MRST
 	default n
 	---help---
 	  Enable the FTL pseudo-filesystem used with the NAND Flash
